style.css

```diff
@@ -484,7 +488,363 @@ footer {
 .liner {
   height: 5px;
   width: 134px;
-  margin: 13px 121px 0 120px;
+  margin-top: 13px;
+  margin-left: 125px;
   background-color: #0d151e;
   border-radius: 10px;
 }
+
+/* -------------------------------------------------------
+ -------desktop version style using media query--------
+------------------------------------------------- */
+
+@media screen and (min-width: 768px) {
+  body {
+    width: 100%;
+  }
+
+  /* header/ navbar media query start */
+
+  .navbar-container {
+    min-height: 72px;
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+  }
+
+  .nav-toggler {
+    display: none;
+  }
+
+  .nav-ul {
+    width: 50%;
+    display: flex;
+    justify-content: space-evenly;
+    align-items: center;
+  }
+
+  .nav-ul li a {
+    font-family: "Inter", sans-serif;
+    font-style: normal;
+    font-weight: 600;
+    font-size: 15px;
+    line-height: 20px;
+    color: var(--color-white);
+  }
+
+  /* header/ navbar media query end */
+
+  /* headline/homepage media query star */
+
+  .section {
+    min-height: 500px;
+    background-color: var(--color-black);
+    background-image: url("./images/desktop-1.svg");
+    background-repeat: no-repeat;
+    background-size: 50%;
+    background-position-x: 100%;
+    background-position-y: 90%;
+  }
+
+  .headline-container {
+    width: 50%;
+    flex-direction: row-reverse;
+  }
+
+  .headline-text {
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+    width: 85%;
+    height: 100%;
+    padding-right: 10%;
+  }
+
+  .headline-text h1 {
+    font-size: 50px;
+  }
+
+  .headline-text h3 {
+    font-size: 25px;
+  }
```

Comment:
- **[OPTIONAL]** Please adjust the line spaces to be similar to the Figma template 👇🏽 

| Figma template | Your design |
|---|---|
| ![image](https://user-images.githubusercontent.com/58642949/214886573-ada0d5a8-c1b5-4f5d-9b79-5add99d2534f.png) | ![image](https://user-images.githubusercontent.com/58642949/214886666-7858ec62-8c4c-4e4b-8253-01e14d5eb151.png) |